### PR TITLE
WSL instructions to move the GCP service account key in the Ubuntu file system

### PR DESCRIPTION
**Pull request by gmanchon**
WSL instructions to move the GCP service account key in the Ubuntu file system

The goal is to avoid this issue :

```
Anonymous caller does not have storage.objects.get access to the Google Cloud Storage object., 401
When students are trying to retrieve the data from their bucket they get a 401 error.
Make sure their JSON credentials are stored within Ubuntu not Windows. The path for GOOGLE_APPLICATION_CREDENTIALS should start with /home  not C: :pray:
```

This also address the issue raised in https://github.com/lewagon/data-setup/issues/76

### CODE REVIEW

File: _partials/gcp_setup_mid.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- Store the service account json file somewhere you'll remember, for example `/Users/YOUR_USER_NAME/Documents/gcp/SERVICE_ACCOUNT_JSON_FILE_CONTAINING_YOUR_SECRET_KEY.json`
```
Comment (zzuziak) [inline _partials/gcp_setup_mid.md:1]: Wouldn't it be better to create a separate instruction for Linux? We don't have `/Users` as absolute path, but `/home/` - less confusion for students on the first day :)

File: macOS.md [hunk 2]
```diff
@@ -1024,32 +1024,47 @@ Now that you have created a `GCP account` and a `project` (identified by its `PR
...
+Now let's verify that the path to your service account json file is correct:
+
 ``` bash
 cat $(echo $GOOGLE_APPLICATION_CREDENTIALS)
 ```
 
-👉 This command should display the content of your service account json file. If it does not, ask for a TA
+👉 This command should display the content of your service account json file. If it does not, ask for a TA 🙏
+
+Your code and utilities are now able to access the ressources of your GCP account.
```
Comment (zzuziak) [inline macOS.md:1065]:
```suggestion
Your code and utilities are now able to access the resources of your GCP account.
```

File: _partials/gcp_setup_end.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Now let's verify that the path to your service account json file is correct:
+
+``` bash
+cat $(echo $GOOGLE_APPLICATION_CREDENTIALS)
+```
+
+👉 This command should display the content of your service account json file. If it does not, ask for a TA 🙏
+
+Your code and utilities are now able to access the ressources of your GCP account.
```
Comment (zzuziak) [inline _partials/gcp_setup_end.md:27]:
```suggestion
Your code and utilities are now able to access the resources of your GCP account.
```

File: _partials/gcp_setup_mid.md (hunk 1 above)
Comment (gmanchon) [reply _partials/gcp_setup_mid.md:1]: I'll do that 👌

Comment (gmanchon) [reply _partials/gcp_setup_mid.md]: @zzuziak just added equivalent storage locations for mac / linux under ~/code/github_nickname/gcp/

**Comment by zzuziak** [comment]
Perfecto, thank you for the quick update! :muscle:

**Pull request merged**